cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -249,6 +281,20 @@ func (i *deployCommand) Exec(cmd *cobra.Command, args []string) (err error) {
 	if hasSource {
 		log.Infof(message.KogitoAppViewDeploymentStatus, i.flags.name, i.flags.Project)
 		log.Infof(message.KogitoAppViewBuildStatus, i.flags.name, i.flags.Project)
+
+	} else if i.flags.fromFile {
+		options.Name = kogitoApp.Name
+		cli, buildCliErr := client.NewClientBuilder().WithBuildClient().Build()
+		if buildCliErr != nil {
+			return buildCliErr
+		}
+
+		_, err := openshift.BuildConfigC(cli).TriggerBuildFromFile(i.flags.Project, userProvidedFile, options)
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		cli, err := client.NewClientBuilder().WithBuildClient().Build()
		if err != nil {
			return err
		}

		_, err = openshift.BuildConfigC(cli).TriggerBuildFromFile(i.flags.Project, userProvidedFile, options)
		if err != nil {
			return err
		}
```

I think it's clearer this way